pkg/interactive-installer/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+FROM lfedge/eve-alpine:9fb9b9cbf7d90066a70e4704d04a6fe248ff52bb AS build
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
+
+ENV BUILD_PKGS curl build-base git ncurses-dev musl-dev python3
+ENV PKGS alpine-baselayout ncurses-dev smartmontools bash mtools strace util-linux e2fsprogs e2fsprogs-extra jq
+
+RUN eve-alpine-deploy.sh
+ENV RUSTUP_HOME=/usr/local
+ENV CARGO_HOME=/usr/local/cargo
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain none
+ENV PATH "$PATH:/usr/local/cargo/bin"
+
+WORKDIR /usr/local/my-installer
+COPY . .
+RUN cargo install --debug --path .
```

Comment:
It occurs to me now, that if we have `network: no` (as we should), then rust might be available from alpine packages or we can use `ADD`. But that won't solve `cargo install`, which requires the network.  Instead, run `cargo vendor` and commit to git, so you can run offline.

Reply:
How did you solve this? I see `network: no` in the build.yml, so you must have gotten it to work. Was it the `vendor_part_*` stuff? What is the process I need to follow if I need to add a new package? Can we document that in the README?